torch/csrc/jit/script/lexer.h

```diff
@@ -134,7 +134,7 @@ struct TokenTrie {
     }
 
     child_chars.emplace_back(*str);
-    child_tries.emplace_back(torch::make_unique<TokenTrie>());
+    child_tries.emplace_back(c10::guts::make_unique<TokenTrie>());
```

Comment:
Can we make make_unique more accessible, not sure why it is in a hidden namespace.

Reply:
`make_unique` is not supposed to be part of the public API, that's why it's not in the top level `c10` namespace. It's an internal tool we use for compatibility with older C++ versions, to be removed next month or so when we can switch to C++14 and just use `std::make_unique`.